### PR TITLE
Chrono on ND, GPS Primary Messages on the CDU.

### PR DESCRIPTION
**Pull request by rcavinash123**
**Summary of Changes**

**Chrono on ND**
1. Added chrono feature on ND. Chrono will display on the ND if the Cap or FO chrono button is pushed.
2. Chrono button behavior is matched with the real life functionality as mentioned below.
    a. Pushing chrono button for the first time will display and start the timer.
    b. Pushing chrono button for the second time will pause the chrono.
    c. Pushing chrono button for the third time will reset and hide the chrono.
3. Both FO and Capt chrono button are linked together since the ND is currently common to both sides.

**GPS Primary Messages On CDU**
1. "GPS Primary" message will now be displayed on the CDU.
2. We can clear the message on CDU by pressing the CLR button on the CDU. This will also clears the message on the ND as well.
3.  "GPS Primary Lost" message will be displayed on the CDU in Amber color. Clearing this message on the CDU will only clear 
      the CDU, it will not clear the ND. As it does IRL.
4. Users can still input any characters on the CDU, while any one of these messages are active. CDU will temporarily hide these 
    messages when any input has been made. CDU will display these messages again once the scratch pad is empty or user has 
    completed his actions.
5. Both the above messages will be displayed on the CDU until the user acknowledges it by pressing the CLR button. The 
    messages will appear again, only when the IRS state changes.

**Screenshots (if necessary)**

1. Chrono on ND (Minutes & Seconds)

Desktop Screenshot 2020 09 26 - 15 47 50 15

2. Chrono on ND (Hours & Minutes)

Desktop Screenshot 2020 09 26 - 16 58 29 58

3. GPS Primary Message on CDU

Desktop Screenshot 2020 09 26 - 18 12 03 53

4.GPS Primary Message (After clearing on CDU)

Desktop Screenshot 2020 09 26 - 18 12 12 10

5.GPS Primary Lost Message on CDU

Desktop Screenshot 2020 09 26 - 19 20 28 49
 

**References**

**Chrono on ND**

Could not find any real life pictures, but i have searched over various pictures on the internet as well as verified the functionality with FFA320 and Toliss. Therefore any IRL pilots feedback will be much appreciated.

image

**GPS Primary Messages**

1. https://hsi.arc.nasa.gov/flightcognition/download/appendix/A320/7_A320_LossDegradationofGPS%20.pdf
image

image

**Additional context**

Discord username (if different from GitHub):rcavinash123

### CODE REVIEW

**Comment by Benjozork** [comment]
Let's wait for #910 to be merged and then you should rebase your changes on it

**Comment by NathanInnes** [comment]
Please remove layout.json & manifest.json from this PR

**Comment by Noxleee** [comment]
Name: Noxlee #7727
Date of testing: Current date 27/09/2020
Version of the sim: 1.8.3.0
PR Tested: #926

Testing technique: Test Flight EGNT -> ENHD
Steps you've made to test the Pull Request:
1. Load up on RW07 in EGNT rdy to start
2. GPS Primary showed up, cleared it on MCDU, also cleared on ND
3. Pressed Chrono and started flight. Chrono ran as described
4. Tested Sim acceleration, also worked
5. Landed in ENHD at 

Overall Rating: Good
Conclusions of the testing: All described functions are working as described, see screenshots

Minutes and seconds:
Microsoft Flight Simulator Screenshot 2020 09 27 - 23 43 56 41

Hours and minutes:
Microsoft Flight Simulator Screenshot 2020 09 28 - 00 19 20 48

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js
Comment (NathanInnes) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js:111]: Tabs
```suggestion
	//CHRONO
	SimVar.SetSimVarValue("L:AUTOPILOT_CHRONO_STATE", "number", 0);
	this.chronoDiv = document.querySelector("#div_Chrono");
	this.text_chrono_prefix = document.querySelector("#text_chrono_prefix");
	this.text_chrono_HRSymbol = document.querySelector("#text_chrono_HRSymbol");
	this.text_chrono_suffix = document.querySelector("#text_chrono_suffix");
	this.IsChronoDisplayed = 0;
```

File: A32NX/layout.json
Comment (NathanInnes) [inline A32NX/layout.json:1]: Remove layout.json. Git CI will perform this action

File: A32NX/manifest.json
Comment (NathanInnes) [inline A32NX/manifest.json:27]: Remove manifest.json as Git CI will perform this action instead.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js
Comment (NathanInnes) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:179]: Use ```const``` not ```var```
```suggestion
		const GPSPrimary = SimVar.GetSimVarValue("L:GPSPrimary", "bool");
		const GPSPrimaryAck = SimVar.GetSimVarValue("L:GPSPrimaryAcknowledged", "bool");	
```

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js
Comment (NathanInnes) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:418]: Use ```const``` not ```var```
```suggestion
			const ack = SimVar.GetSimVarValue("L:GPSPrimaryAcknowledged", "bool");	
			const isGPSPrimaryMessageDisplayed = SimVar.GetSimVarValue("L:GPSPrimaryMessageDisplayed", "bool");				
```

**Review by NathanInnes — CHANGES_REQUESTED** [review]

**Comment by FlyingCarpet7** [comment]
Your Chrono on ND logic is all correct!
/IRL pilot

**Comment by rcavinash123** [comment]
> Your Chrono on ND logic is all correct!
> /IRL pilot

Yes i would wait, until the PR request for separate ND is merged.

**Comment by wpine215** [comment]
@rcavinash123 heads up, PR request for separate ND has now been merged.

**Comment by AdenFlorian** [comment]
i merged master into this, fixed conflicts and bugs from the merge. now the PR code needs to be changed to account for separatated NDs

**Comment by AdenFlorian** [comment]
I confirmed with danbw9 in discord that the chrono's for FO and CPT are completely independent of eachother.

I know that this displays were still forced in sync when this went up, just needed to check before I fixed it to be independent.

**Comment by AdenFlorian** [comment]
LGTM!

**Comment by FlyingCarpet7** [comment]
Nice work with the Chrono on ND! 
Maybe you can tweak the font size and brightness a bit? Real size seems to match rest of the numbers on ND, Eg GS numbers. Brightness of the chrono in the mod seems to be lower than the rest on the ND.

Chrono 2
Chrono real

**Pull request merged**
